src/frame.js

```diff
@@ -101,7 +101,9 @@ if (mmAppear) {
     provider = new MetaMaskProvider(new Connection())
     provider.isMetaMask = true
     provider._metamask = {
-      isUnlocked: () => true
+      isUnlocked: () => new Promise((resolve) => {
+        resolve(true)
+      })
```

Comment:
Can probably make this a one-liner:
`isUnlocked: () => new Promise((resolve) => resolve(true))`

Reply:
@goosewobbler updated